### PR TITLE
Adapted GUI to Composite Pattern and functional changes

### PR DESCRIPTION
**Pull request by francescofact**
Adapted GUI to Composite Pattern and functional changes
Restaurant list is now public in index

### CODE REVIEW

File: monolith/templates/navbar/admin.html [hunk 1]
```diff
@@ -1,13 +1,11 @@
...
-        </li>
-        <li class="nav-item">
-            <a class="nav-link" href="/report_positive">Report positive</a>
-        </li>
-    </ul>
-</div>
\ No newline at end of file
+
+  <li class="nav-item">
+    <a class="nav-link" href="#">Home
```
Comment (vincenzopalazzo) [inline monolith/templates/navbar/admin.html:3]: maybe is better to have an href="/"?

**Review by vincenzopalazzo — CHANGES_REQUESTED** [review]
I like it, only a small comment :-)

File: monolith/templates/navbar/admin.html (hunk 1 above)
Comment (francescofact) [reply monolith/templates/navbar/admin.html:3]: yeah, totally agree, it doesn't make sense the "#"

Comment (vincenzopalazzo) [reply monolith/templates/navbar/admin.html]: Thanks, marked as resolved :-)

**Comment by vincenzopalazzo** [comment]
@vincenzopalazzo review this PR

**Pull request merged**